src/FilamentDevelopersLogin.php

```diff
@@ -19,6 +19,7 @@ public function login(Panel $panel, FilamentDeveloperLoginsPlugin $plugin, strin
 
         if ($panel->auth()->check()) {
             $panel->auth()->logout();
+            session()->forget('password_hash_'.$panel->getAuthGuard());
```

Comment:
Is this really what caused the route not found bug? Can you explain why this fixes it?